.gitignore

```diff
@@ -656,5 +655,5 @@ InitTestScene*.unity
 # Ionide (cross platform F# VS Code tools) working folder
 
 # End of https://www.toptal.com/developers/gitignore/api/unity,csharp,rider,vscode,visualstudio
-DRUNK/Library/AnnotationManager
```

Comment:
Why was this removed from the `.gitignore`?

Reply:
The library folder is already ignored as a whole, this iwas redundant